Samples/iOS-Swift/iOS-SwiftUITests/LaunchUITests.swift

```diff
@@ -67,8 +67,8 @@ class LaunchUITests: XCTestCase {
         let app = XCUIApplication()
         app.navigationBars["iOS_Swift.SecondarySplitView"].buttons["Root ViewController"].waitForExistence("SplitView not loaded.")
         
-        // This validation is currently not working on iOS 10.
-        if #available(iOS 11.0, *) {
+        // This validation is currently not working on iOS 12 and iOS 10.
```

Comment:
I created an issue for enabling the test again https://github.com/getsentry/sentry-cocoa/issues/2313